test/unit/orm-adapters-memory-test.js

```diff
@@ -0,0 +1,14 @@
+import test from 'ava';
+import { MemoryAdapter } from '../../lib/data/orm-adapters/memory';
```

Comment:
Shouldn't it be `import MemoryAdapter from '../../lib/data/orm-adapters/memory';`